src/commands/finalize.ts

```diff
@@ -0,0 +1,35 @@
+import {flags} from '@oclif/command'
+import PercyCommand from './percy-command'
+import BuildService from '../services/build-service'
+
+export default class Finalize extends PercyCommand {
+  static description = 'finalize a build'
+  static hidden = false
+
+  static flags = {
+    all: flags.boolean({char: 'a'}),
+  }
+
+  static examples = [
+    '$ percy-agent finalize --all\n' +
+    '[percy] Finalized parallel build.',
+  ]
+
+  buildService: BuildService = new BuildService()
+
+  async run() {
+    const {flags} = this.parse(Finalize)
+
+    if (this.percyEnvVarsMissing()) { return }
+
+    if (!process.env.PERCY_PARALLEL_NONCE) {
```

Comment:
I think at some point we could make this generic and have a function that takes in all the required env vars, outputs to the log if they're missing and exits. But for now this'll do.